packages/fixtures/fixtures.coffee

```diff
@@ -44,3 +45,25 @@ do ->
 
     'createOrg': (attributes) ->
       Organizations.insert attributes
+
+    'addDataset': (attributes)->
+      dataset = new Dataset()
+      dataset.set(attributes)
+      (Meteor.wrapAsync((callback)->
```

Comment:
Why can't it be synchronous without this? Does dataset.save not run synchronously if you leave out the callback?